src/libraries/Directory.Build.targets

```diff
@@ -139,6 +139,12 @@
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.TargetFramework" Version="$(MicrosoftDotNetBuildTasksTargetFrameworkVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Fixup arcade test references by removing xunit" -->
+    <PackageReference Remove="xunit" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
+    <PackageReference Remove="xunit.assert" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
+  </ItemGroup>
```

Comment:
Remove this. We don't use Arcade's test framework: https://github.com/dotnet/runtime/blob/80ffe5c713b87524aa87085cff81d09daf8c65b4/src/libraries/Directory.Build.props#LL3C3-L3C3